go/coordinator/cmd/coordinator/cmd.go

```diff
@@ -30,14 +30,15 @@ func init() {
 	flag.GRPCAddr(Cmd, &conf.GrpcConfig.BindAddress)
 
 	// System Catalog
-	Cmd.Flags().StringVar(&conf.SystemCatalogProvider, "system-catalog-provider", "memory", "System catalog provider")
-	Cmd.Flags().StringVar(&conf.Username, "username", "root", "MetaTable username")
-	Cmd.Flags().StringVar(&conf.Password, "password", "", "MetaTable password")
-	Cmd.Flags().StringVar(&conf.Address, "db-address", "127.0.0.1", "MetaTable db address")
+	Cmd.Flags().StringVar(&conf.SystemCatalogProvider, "system-catalog-provider", "database", "System catalog provider")
```

Comment:
are we sure want to change the default?

Reply:
yes because memory is not Postgres and I want to have local end to end set up to include postgres

Reply:
Then should we just remove memory - is it used?

Reply:
ok, can we add some tooling to clear/nuke postgres in that case as a follow up?

Reply:
3 parts of this:
1. unit tests: should clean up the state - add setup and teardown (I can add this, very easy)
2. local tests: scripts to nuke, for now the easiest thing is to just drop and re-create the db
3. docker end to end test: there is a refresh button in tilt UI, one click should do the work

Reply:
Docker restart should not remove state. We should use volumes correctly and have a script provided for People to reset the db

Reply:
I do not use volume for the local docker setup as of now. can go add that later if we want persistence.

Reply:
We should use volumes. Lets do the right thing from the get go please!